internal/datacoord/segment_manager.go

```diff
@@ -329,13 +329,17 @@ func isGrowing(segment *SegmentInfo) bool {
 	return segment.GetState() == commonpb.SegmentState_Growing
 }
 
-func (s *SegmentManager) genExpireTs(ctx context.Context) (Timestamp, error) {
+func (s *SegmentManager) genExpireTs(ctx context.Context, isImported bool) (Timestamp, error) {
 	ts, err := s.allocator.allocTimestamp(ctx)
 	if err != nil {
 		return 0, err
 	}
 	physicalTs, logicalTs := tsoutil.ParseTS(ts)
 	expirePhysicalTs := physicalTs.Add(time.Duration(Params.DataCoordCfg.SegAssignmentExpiration.GetAsFloat()) * time.Millisecond)
+	// for imported segment, clean up ImportTaskExpiration
+	if isImported {
+		expirePhysicalTs = physicalTs.Add(time.Duration(Params.RootCoordCfg.ImportTaskExpiration.GetAsFloat()) * time.Millisecond)
```

Comment:
Should be `time.Second` here.

Also I found it inconsistent that sometimes we have both `second` and `millisecond` as time measurement in a lot of places in our `milvus.yaml` file